### PR TITLE
Update resources.json

### PR DESCRIPTION
**Pull request by ThomasKomenda**
Added sessions links to all 6 tracks

Fixes #

## Proposed Changes

  -add the category "Session Link" to all 6 tracks at the beginning of each track overview. 
  -replace placeholders in Newcomer and ACME tracks with "Session Link" and "Track Content"

### CODE REVIEW

**Review by kaimeinke — APPROVED** [review]
Approved.

**Review by kaimeinke — APPROVED** [review]

**Comment by kaimeinke** [comment]
Please merge this one @oceanByte / @moritzkirstein

**Comment by moritzkirstein** [comment]
@kaimeinke you have write access now to this repository and will be able to merge after approval from now on.

**Pull request merged**
